storage/node.go

```diff
@@ -460,9 +460,6 @@ func (n *NodeID) Split(prefixBytes, suffixBits int) ([]byte, *Suffix) {
 	if n.PrefixLenBits == 0 {
 		return []byte{}, EmptySuffix
 	}
-	a := make([]byte, prefixBytes)
```

Comment:
I think this was missed in previous refactorings. Previously not all the methods returned copies so prefix was copied here.